evaluation/appraisal/ariana/capital_appraisal.md

```diff
@@ -1,6 +1,6 @@
 #lang abe/attribution2 capital-analysis
 
-* Old Abe
+* Old Abe [100%]
```

Comment:
Looks like it doesn't matter that these numbers are at the beginning vs the end eh?